migrations/sqls/20201028204537-followups-up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS followUps (
```

Comment:
This table should have a `clientId` column that is a foreign key to the `clients` table.